src/stories/Dropdown.stories.js

```diff
@@ -31,6 +31,7 @@ FlatArrayExample.args = {
   onChange: (option) => {
     console.log('You selected ', option.label);
   },
+  isClearable: true,
```

Comment:
create an example specifically for clearing the value

Reply:
a story is added for the clear button